docs/config.json

```
@@ -97,6 +97,11 @@
               "slug": "/deploy-a-cluster/teleport-enterprise/hsm/",
               "forScopes": ["enterprise"]
             },
+            {
+              "title": "Google KMS",
```

```suggestion
              "title": "Google Cloud KMS",
```

Suggestion to use the proper product name for maximum discoverability.